cime_config/testdefs/testlist_clm.xml

```diff
@@ -462,13 +462,43 @@
       <option name="wallclock">00:20:00</option>
     </options>
   </test>
-  <test name="ERP_D_Ld10_P36x2" grid="f10_f10_mg37" compset="IHistClm51BgcCrop" testmods="clm/ciso_decStart">
+  <test name="ERP_D_Ld10" grid="f10_f10_mg37" compset="IHistClm50BgcCrop" testmods="clm/ciso_decStart_GU_LULCC">
+    <machines>
+      <machine name="izumi" compiler="nag" category="aux_clm"/>
+    </machines>
+    <options>
+      <option name="wallclock">00:40:00</option>
+      <option name="comment"  >Transient case with isotopes with a december start, and Gross Unrepresented Land Use and Land Cover change on, default dataset with GU_LULCC==0</option>
+    </options>
+  </test>
+  <test name="ERP_D_Ld10_P36x2" grid="f10_f10_mg37" compset="IHistClm50BgcCrop" testmods="clm/ciso_decStart_GU_LULCC">
     <machines>
       <machine name="cheyenne" compiler="intel" category="aux_clm"/>
     </machines>
     <options>
-      <option name="wallclock">00:20:00</option>
-      <option name="comment"  >Transient case with isotopes with a december start</option>
+      <option name="wallclock">00:40:00</option>
+      <option name="comment"  >Transient case with isotopes with a december start, and Gross Unrepresented Land Use and Land Cover change on, default dataset with GU_LULCC==0</option>
+    </options>
+  </test>
+  <test name="ERP_D_Ld10" grid="f09_f09_mg17" compset="IHistClm50BgcCrop" testmods="clm/f09_dec1990Start_GU_LULCC">
+    <machines>
+      <machine name="cheyenne" compiler="intel" category="aux_clm"/>
+      <machine name="cheyenne" compiler="gnu" category="aux_clm"/>
+      <machine name="izumi" compiler="nag" category="aux_clm"/>
+    </machines>
+    <options>
+      <option name="wallclock">00:59:00</option>
```

Comment:
@slevisconsulting to make sure the new code was working we added more tests than we really need to for ongoing development. So we should shorten the list to maybe one or two. But, since we need to move this over to CTSM5.2, let's leave all these tests in place, but remove most of them in a future tag on CTSM main-dev. 

Let's make an new issue about paring the number of these GULU tests down, and we can figure out which ones to keep there.

Reply:
#1968 takes care of this comment.